.golangci.yml

```diff
@@ -19,3 +19,11 @@ linters:
     - prealloc
     - scopelint
     - gocritic
+
+issues:
+  exclude-use-default: false
+  exclude:
+    - G104 # Errors unhandled
+    - G103 # Use of unsafe calls should be audited
+    - G204 # Subprocess launched with variable
+    - G304 # Potential file inclusion via variable
```

Comment:
Can you end the file with a newline?

Reply:
Newline is required at the end of file. Otherwise you'll get `File is not `goimports`-ed (goimports)`

Reply:
This file (`.golangci.yml`) does not end in a newline.  Can you end a newline here?